gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,650 @@
+
+. Run the edited command from your terminal window.
++
+When successful, the command produces a `registration.yaml` in your `./flex-registration` directory and provides the following message in your terminal window:
++
+----
+Starting registration, please be patient.
+Registration completed, the configuration files were written 
+in directory "/registration". For security, modify the file permissions 
+to restrict production scenario access to the user running flex.
+----
++
+Note that the message in the terminal window refers to a `/registration` directory that resides within the Docker container. 
++
+Use `sudo` if you encounter file permission issues when running this command. If your token expires before you run the command, a `400 Bad Request` error occurs, so you need to refresh the Runtime Manager page to generate a new token and run the modified command that appears on the page.
+
+[[create-repo]]
+== Step 3: Create a Repository for the Helm Chart
+
+Add the Flex Gateway chart repository. Note that you can get more information about the properties of the Helm chart for Flex Gateway in https://artifacthub.io/packages/helm/flex-gateway/flex-gateway[ArtifactHUB^].
+
+. Add a Helm repository named `flex-gateway` for your Helm chart: 
++
+[source,helm]
+----
+helm repo add flex-gateway https://flex-packages.anypoint.mulesoft.com/helm
+----
++
+The command either adds the repository or skips this process if a Helm repository with that name exists already on your machine:
++
+* If the repository is new, the command returns the following message:
++
+----
+"flex-gateway" has been added to your repositories
+----
++
+* If the repository exists already, the command returns the following message:
++
+----
+"flex-gateway" already exists with the same configuration, skipping
+----
+
+. Run `helm repo up`.
++
+The command returns the following message:
++
+----
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "flex-gateway" chart repository
+Update Complete. ⎈Happy Helming!⎈
+----
++
+If you have more than one Helm repository on your machine, the message in your terminal window lists all of them.
+
+[[deploy-flex]]
+== Step 4: Deploy and Connect Flex Gateway
+
+Use Helm to deploy Flex Gateway to a node in your cluster and connect it to Anypoint Platform. After deploying, use Runtime Manager to verify that the gateway is present and connected to Anypoint Platform. 
+
+
+Before you begin: 
+ 
+Confirm that your Kubernetes cluster is available by following the steps in <<check-cluster>>. Review <<k8-cluster-reqs>> if you have not done so already. These prerequisites help avoid deployment errors. 
+
+After confirming that your Kubernetes cluster is ready, complete the following steps:
+
+. Run the Helm command for deploying your gateway and connecting it to Anypoint Platform. 
```

Comment:
should "Helm" be lowercase/mono in this case?

Reply:
it's ok to stay as-s